docs/csharp/language-reference/operators/null-conditional-operators.md

```diff
@@ -5,13 +5,23 @@ ms.prod: .net
 ms.technology: 
   - "devlang-csharp"
 ms.topic: "article"
+dev_langs: 
+  - "csharp"
+  - "vb"
+helpviewer_keywords: 
+  - "null-conditional operators [C#]"
+  - "null-conditional operators [Visual Basic]"
+  - "?. operator [C#]"
+  - "?. operator [Visual Basic]"
+  - "?[] operator [C#]"
+  - "?[] operator [Visual Basic]"
```

Comment:
Or it should be `?()` for Visual Basic?

Reply:
That's a good question.  I'm going to ask @KathleenDollard or @rpetrusha to weigh in on that.

I looked through the VB language spec, and I could not find a definitive answer.